package.json

```diff
@@ -21,7 +21,7 @@
     "jsonld": "^3.2.0",
     "lodash.isplainobject": "^4.0.6",
     "prop-types": "^15.6.2",
-    "react-admin": "^3.0.0"
+    "react-admin": "^3.11"
```

Comment:
It shouldn't be needed. Why this change?

Reply:
react-admin 3.0.0 is from Nov 20, 2019, several corrections and bugfix was implemented since then

https://github.com/marmelab/react-admin/compare/v3.0.0...v3.11.4

Reply:
Sure but it just means that all the versions are supported starting from 3.0, since there is no breaking change.